project/Build.scala

```diff
@@ -303,6 +303,7 @@ object ScaldingBuild extends Build {
       "com.twitter" %% "bijection-macros" % bijectionVersion,
       "com.twitter" %% "chill" % chillVersion,
       "com.twitter" %% "chill-algebird" % chillVersion,
+      "com.twitter.elephantbird" % "elephant-bird-cascading2" % elephantbirdVersion,
```

Comment:
nope. We don't want to make this a dependency of scalding-core. It is a dependency of scalding-commons. This means you should move the LZO related text stuff into scalding-commons, but the other stuff can stay here.